src/classes/class-typography.php

```diff
@@ -0,0 +1,381 @@
+                    );
+                }
+            }
+
+            // Global custom Typography.
+            if ( $this->is_exist( $global_typography ) && $this->is_exist( $global_typography['ghostkit_typography'] ) ) {
+                $typography_prepeare_styles = $this->get_typography_values( $global_typography['ghostkit_typography'], $typography_prepeare_styles );
+            }
+
+            // Local custom Typography.
+            if ( $is_single || $is_admin_editor ) {
+                $meta_typography = get_post_meta( $post_id, 'ghostkit_typography', true );
+                $typography_prepeare_styles = $this->get_typography_values( $meta_typography, $typography_prepeare_styles );
+            }
+
+            // Collect all the styles for further transfer to the inline file on the edit or front page.
+            foreach ( $typography_prepeare_styles as $typography_prepeare_style ) {
+                if ( ( $this->is_exist( $typography_prepeare_style['output'] ) && is_array( $typography_prepeare_style['output'] ) ) &&
+                     ( $this->is_exist( $typography_prepeare_style['style-properties'], 'font-family' ) ||
+                         $this->is_exist( $typography_prepeare_style['style-properties'], 'font-size' ) ||
+                         $this->is_exist( $typography_prepeare_style['style-properties'], 'font-weight' ) ||
+                         $this->is_exist( $typography_prepeare_style['style-properties'], 'line-height' ) ||
+                         $this->is_exist( $typography_prepeare_style['style-properties'], 'letter-spacing' )
+                     ) ) {
+                    foreach ( $typography_prepeare_style['output'] as $output ) {
+                        if ( $this->is_exist( $output['selectors'] ) ) {
+                            $typography_styles = '';
+                            $typography_styles .= $output['selectors'] . '{';
+
+                            if ( $this->is_exist( $typography_prepeare_style['style-properties'], 'font-family' ) ) {
+                                $typography_styles .= 'font-family: ' . $typography_prepeare_style['style-properties']['font-family'] . ';';
+                            }
+                            if ( $this->is_exist( $typography_prepeare_style['style-properties'], 'font-size' ) ) {
+                                $typography_styles .= 'font-size: ' . $typography_prepeare_style['style-properties']['font-size'] . ';';
+                            }
+                            if ( $this->is_exist( $typography_prepeare_style['style-properties'], 'font-weight' ) ) {
+                                $font_weight = $typography_prepeare_style['style-properties']['font-weight'];
+                                if ( false !== strpos( $font_weight, 'i' ) ) {
+                                    $font_weight = str_replace( 'i', '', $font_weight );
+                                    $typography_styles .= 'font-style: italic;';
```

Comment:
Let's add `font-style: normal;` when selected non-italic font. The problem appears when we add italic font on `body` and normal font on `headings`. Headings appear as italic.